_content/references.md

```diff
@@ -1,5 +1,170 @@
+| Agora Real-time Messaging SDK, Agora RTM SDK | RTM SDK                             | Used to create a stable messaging mechanism for real-time messaging scenarios. Successor to the Agora Signaling SDK. |
+| Agora SDK                                    | Agora SDK                           | A generic term for referring to any Agora SDK. Preference is to specify the name of the SDK. |
+| Agora SD-RTN™                                | SD-RTN™                             | "Software-defined Real-time Network". Must appear with trademark symbol, preferred version of term. |
+| Agora Video SDK                              | Video SDK                           | Used to embed video calling features into an app.            |
+| Agora RTC Web SDK                            | Web SDK                             | Uses APIs in the web browser to establish connections and control the communication and live broadcast services. |
+| Agora Interactive Whiteboard                 | Interactive Whiteboard              | Provides a simple collaboration platform on a virtual whiteboard where users from different locations can draw, annotate, and share PDF documents. |
+| Agora, Inc.                                  | Agora                               | Official company name.                                       |
+| Agora.io                                     | Agora                               | Variant of company name. Not used in technical documentation, with the exception of copyright statements (for example, "© 2016 Agora.io. All rights reserved.") |
+
+#### A
+
+- **add-on** (noun, adjective), **add on** (verb)
+- **Android device**
+- **app** (lowercase except when beginning a sentence or used as a proper noun)
+- **audio sample**: Use instead of *audio capture*.
+- **audio sample rate**
+
+#### B
+
+- **backend**: Use instead of *back-end* or *back end*.
+- **beta**
+- **bitrate**
+- **Boolean** (uppercase when referring to the datatype)
+- **Bluetooth**
+- **bridging header file**
+
+#### C
+
+- **case-sensitive**
+
+#### D
+
+- **data packet**
+- **dual-stream mode**
+- **dual stream** (when not modifying a noun)
+
+#### E
+
+- **email**
+- **earlier**: Use instead of *lower* when describing a version.
+- **extract**: Use instead of *unarchive* or *uncompress*.
+
+#### F
+
+- **filename**
+- **fill in; fill out**: Use *fill in* when referring to entering information in individual fields. Use *fill out* when referring to completing an entire form.
+- **front camera**
+
+#### G
+
+- **get**: Use instead of *fetch* or *retrieve*.
+
+#### H
+
+- **HTML5**: Use instead of *H5*.
+
+#### I
+
+- **incorrect**: Use instead of *wrong*.
+- **image enhancement**
+- **internet**
+
+#### J
+
+- **join a channel**: Use instead of *enter a channel*.
+- **just**: Avoid. Usually, *just* is a filler word that you can delete without affecting your meaning.
+
+#### L
+
+- **later**: Use instead of *higher* when describing a version.
+- **live stream** (verb or noun)
```

Comment:
May need more explanation on when "live stream" applies, instead of "live streaming". Because the product is "Interactive Live Streaming Premium", I tend to use "live streaming" to refer to the scenario.